modules/spring-config/src/main/resources/META-INF/mule.xsd

```diff
@@ -3179,22 +3192,8 @@
             <xsd:complexContent>
                 <xsd:extension base="abstractRoutingMessageProcessorType">
                     <xsd:sequence>
-                        <xsd:element name="custom-aggregation-strategy" type="aggregationStrategyType" minOccurs="0" maxOccurs="1">
-                            <xsd:annotation>
-                                <xsd:documentation>
-                                    Allows customizing the logic used for aggregation the events back together.
-                                </xsd:documentation>
-                            </xsd:annotation>
-                        </xsd:element>
-                        <xsd:group ref="messageProcessorOrMixedContentMessageProcessor" minOccurs="2" maxOccurs="unbounded"/>
+                        <xsd:group ref="routeOrProcessorChain" minOccurs="2" maxOccurs="unbounded"/>
```

Comment:
I think routes should have a mandatory id. @pablolagreca what's your opinion on this?